exporter.go

```diff
@@ -51,7 +51,8 @@ func (e *Exporter) buildConfig(app *AppConfig, workingDir, stack string) (*engin
 		Env:        mapToEnv(mergeMaps(env, app.RunningEnv, app.Env)),
 		Image:      stack,
 		WorkingDir: workingDir,
-		Entrypoint: []string{"/packs/launcher", app.Command},
+		Entrypoint: []string{"/packs/launcher"},
```

Comment:
Test needs to be updated: https://github.com/buildpack/forge/blob/master/exporter_test.go#L76